tests/test_mycroft_skills_manager.py

```diff
@@ -67,9 +67,9 @@ def _mock_log(self):
         self.log_mock = log_patch.start()
 
     def _mock_skills_json_path(self):
-        expanduser_patch = patch('msm.skill_state.expanduser')
-        self.addCleanup(expanduser_patch.stop)
-        self.skills_json_path_mock = expanduser_patch.start()
+        savedatapath_patch = patch('msm.skill_state.BaseDirectory.save_data_path')
```

Comment:
I'm not familiar yet with mock patching, not sure how to do this correctly. Tests seem to fail on it

Reply:
easiest may be to set `msm.SKILL_STATE_PATH = self.skills_json_path` in `setUp()` and restore in `tearDown()` instead of using the mock...(can't find a way to mock a string) or similar...

Reply:
I cherry-picked your commit. It makes sense, just return a string in a function and mock that function, it seems good to me :+1: